src/styles/blocks/header.scss

```diff
@@ -0,0 +1,12 @@
+.container {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+  align-items: center;
+}
```

Comment:
this style doesn't belong to header. it would be great to create separate scss file for it

Reply:
but i have header with class container
where should i move it then?
normalize.scss or split block into logo and container?